src/test/java/com/sda/training/pages/BasePage.java

```diff
@@ -0,0 +1,14 @@
+package com.sda.training.pages;
+
+import com.sda.training.DriverManager;
+import org.openqa.selenium.remote.RemoteWebDriver;
+import org.openqa.selenium.support.PageFactory;
+
+public abstract class BasePage {
+    public final RemoteWebDriver driver;
+
+    BasePage() {
+        this.driver = DriverManager.DRIVER;
+        PageFactory.initElements(this.driver, this);
+    }
+}
```

Comment:
I think below method will be useful to avoid code duplication in extending classes:
```suggestion
void waitUntil(ExpectedCondition condition) {
        new WebDriverWait(driver, TIMEOUT_IN_MS).until(condition);
}
```